File: CrazyBooks_Final/CrazyBooks.Web/wwwroot/Scripts/Models/Book.js
class Book extends Entity
{
    constructor(json)
    {
        super( json);

        if (json)
        {
            this.Title = json.title;
            this.Author = json.author;
            this.PublicationDate = json.publicationDate;
            this.Edition = json.edition;
        }
        else
        {
            this.Title = "";
            this.Author = "";
            this.PublicationDate = null;
            this.Edition = 0;
        }
    }
}

File: CrazyBooks_Final/CrazyBooks.Lib/Models/Book.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Text;

namespace CrazyBooks.Lib.Models
{
    public class Book : Entity
    {
        public string Title { get; set; }
    
        public string Author { get; set; }

        public DateTime? PublicationDate { get; set; }

        public int Edition { get; set; }
        [JsonIgnore]
        public ICollection<Lend> Lends { get; set; }
    }
}
